fixtures/s3/ceph-rados/docker-compose.yml

```diff
@@ -0,0 +1,141 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '3.8'
+
+networks:
+  cluster-net:
+    external: true
+    name: ceph-cluster-net
+
+services:
+  mon1:
+    image: ceph/daemon:latest
+    command: "mon"
+    environment:
+      MON_IP: 172.18.0.2
+      CEPH_PUBLIC_NETWORK: 172.18.0.0/16
+      LANG: en_US.utf8
+      TZ: UTC
+    volumes:
+      - ~/volumes_data/ceph/ceph:/etc/ceph
```

Comment:
Please don't use local path like this. Our CI might not have those file.

You can define a volume like:

```yaml
volumes:
  ceph-data:
```

And use it here:

```yaml
    volumes:
      - ceph-data:/var/lib/ceph/
```

To mount local files:

This path will relative to the location of docker-compose file.

```yaml
    volumes:
      - ./nginx.conf:/etc/nginx/nginx.conf
```

Reply:
Yes, if we do not run the `docker-compose up -d mon1 mgr` command, the CI mechine does not have those files.

<img width="707" alt="image" src="https://github.com/apache/opendal/assets/63448884/f9c42ebe-ac4d-4c84-86d7-633f61ff9ea8">



But, it seems that after running the `docker-compose up -d mon1 mgr` command, we will have those files in the mechine that executes the CI.

<img width="776" alt="image" src="https://github.com/apache/opendal/assets/63448884/29ce0193-7eff-43d0-ac52-d50630bc61b6">

Reply:
I don't understand this comment. My understanding is that Ceph automatically creates those files, so there's no need for us to do anything.

We can specify the start order by set `depends` in docker-compose file.